cypress/features/regression/experimental/dateInput.feature

```diff
@@ -52,34 +52,34 @@ Feature: Experimental Date Input component
 
   @positive
   Scenario: Open dayPickerDay via click on input
-    Given I open "Experimental Date Input Test" component page "default"
-    When I click dateInput
+    Given I open default "Experimental Date Input Test" component in noIFrame with "dateInput" json from "experimental" using "default" object name
+    When I click dateInput in noIframe
```

Comment:
we have only in this line `in noIframe`, so we need to use everywhere with `in noiFrame` or remove it from this line and replace it with proper step